### ISSUE TITLE
Add "Upgrade" procedure to README.md

### ISSUE THREAD

**Issue by jrogers1126** [Happyr/NewRelic-Disk-Monitor-plugin#4]
its great that 0.20 is available, but it is not clear what the best method of updating is.

**Comment by Nyholm** [comment]
You are correct. I will give this plugin in some love soon. The only BC breaking stuff from 0.1 to 0.2 is the rename of the plugin. Have a look here: https://github.com/Happyr/NewRelic-Disk-Monitor-plugin/compare/0.1.0...0.2.0

Any PRs are welcome.
